<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpDescriptionEditor</class>
 <widget class="QWidget" name="ExpDescriptionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mntGrpTab">
      <attribute name="title">
       <string>Measurement Group</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Active Measurement Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="activeMntGrpCB"/>
         </item>
         <item>
          <widget class="QToolButton" name="mntGrpOptionBT">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MntGrpChannelEditor" name="channelEditor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Snapshot Group</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Sardana Elements</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="SardanaElementTreeWidget" name="sardanaElementTree"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>External (Tango)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="TaurusModelSelectorTree" name="taurusModelTree"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>(Drag elements from the above selectors and drop them at the bottom list)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TaurusModelList" name="preScanList"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Storage</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>246</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>File Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="filenameLE"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="pathLE"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Data compression</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="compressionCB">
         <item>
          <property name="text">
           <string>nowhere</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>for all datasets</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>for datasets of rank 1 or more</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>for datasets of rank 2 or more</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>for datasets of rank 3 or more</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>245</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SardanaElementTreeWidget</class>
   <extends>QMainWindow</extends>
   <header>taurus.qt.qtgui.extra_sardana</header>
  </customwidget>
  <customwidget>
   <class>TaurusModelList</class>
   <extends>QListView</extends>
   <header>taurus.qt.qtgui.panel</header>
  </customwidget>
  <customwidget>
   <class>MntGrpChannelEditor</class>
   <extends>QMainWindow</extends>
   <header>taurus.qt.qtgui.extra_sardana</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusModelSelectorTree</class>
   <extends>TaurusWidget</extends>
   <header>taurus.qt.qtgui.panel</header>
  </customwidget>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
